### DISCUSSION TITLE
Update version using pip

### DISCUSSION THREAD

**Discussion by LeifParr** [petrobras/ross#920] [Q&A] [answered]
Hi. I discovered that I was still using ross version 1.0.1, while the latest tag here at github I believe is 1.2.0. So I tried to upgrade through PIP. Does anyone know why I couldn't get the latest version that way? 

Here are my pip version results:

C:\WINDOWS\system32>python -m pip index versions ross-rotordynamics
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
ross-rotordynamics (1.0.1)
Available versions: 1.0.1, 1.0.0, 0.4.1, 0.4.0, 0.3.3, 0.3.2, 0.3.1, 0.3.0, 0.2.0, 0.1.7, 0.1.6, 0.1.2, 0.1.1, 0.1.0, 0.0.2, 0.0.1
  INSTALLED: 1.0.1
  LATEST:    1.0.1

**Comment by raphaeltimbo** [comment]
Hi @LeifParr.
What is your python version?
The 1.2.0 version requires python 3.7+.
Since we are now using pint 0.18 which requires python 3.7+, we can no
longer support python 3.6.

**Comment by LeifParr** [comment, marked as answer]
Hi again. I use python 3.10, so I don't think that could be the issue. Pip
is also up to date. Any other thoughts on how to move forward with regular
pip updates?

**Reply by raphaeltimbo** [reply]
Hi @LeifParr, the problem was that python 3.10 was not supported in those releases.
I have just made a v1.3.0 release with python 3.10 support. 
If you could please test that and see if everything is working ok.

**Reply by LeifParr** [reply]
Excellent, now it went right through with PIP!

python -m pip install --upgrade ross-rotordynamics
Requirement already satisfied: ross-rotordynamics in c:\program files\python310\lib\site-packages (1.0.1)
Collecting ross-rotordynamics
  Downloading ross_rotordynamics-1.3.0-py3-none-any.whl (185 kB)
     ---------------------------------------- 186.0/186.0 kB ? eta 0:00:00
Requirement already satisfied: plotly>=4.7.0 in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (5.10.0)
Requirement already satisfied: pandas>=0.23 in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (1.5.0)
Requirement already satisfied: xlrd in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (2.0.1)
Requirement already satisfied: scipy in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (1.9.1)
Requirement already satisfied: numpy in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (1.23.3)
Requirement already satisfied: pint>=0.18 in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (0.19.2)
Requirement already satisfied: toml>=0.10.1 in c:\program files\python310\lib\site-packages (from ross-rotordynamics) (0.10.2)
Requirement already satisfied: pytz>=2020.1 in c:\program files\python310\lib\site-packages (from pandas>=0.23->ross-rotordynamics) (2022.2.1)
Requirement already satisfied: python-dateutil>=2.8.1 in c:\program files\python310\lib\site-packages (from pandas>=0.23->ross-rotordynamics) (2.8.2)
Requirement already satisfied: tenacity>=6.2.0 in c:\program files\python310\lib\site-packages (from plotly>=4.7.0->ross-rotordynamics) (8.1.0)
Requirement already satisfied: six>=1.5 in c:\program files\python310\lib\site-packages (from python-dateutil>=2.8.1->pandas>=0.23->ross-rotordynamics) (1.16.0)
Installing collected packages: ross-rotordynamics
  Attempting uninstall: ross-rotordynamics
    Found existing installation: ross-rotordynamics 1.0.1
    Uninstalling ross-rotordynamics-1.0.1:
      Successfully uninstalled ross-rotordynamics-1.0.1
Successfully installed ross-rotordynamics-1.3.0
